doc/source/user_guide/indexing.rst

```diff
@@ -1625,7 +1625,7 @@ Missing values
    idx2
    idx2.fillna(pd.Timestamp('2011-01-02'))
 
-Set / reset index
+Set / reset Index
```

Comment:
Should index be in caps here (same for other cases in this PR)? I think this is talking about something more general than the Index class specifically (note it's lowercased in the actual text below).